__tests__/data/salesforce_router_metadata_input.json

```
@@ -0,0 +1,98 @@
+{
+  "batchedRequest": {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://ap15.salesforce.com/services/data/v50.0/sobjects/Lead",
+    "headers": {
+      "Content-Type": "application/json",
+      "Authorization": "Bearer 00D2v000002lXbX!ARcAQJBSGNA1Rq.MbUdtmlREscrN_nO3ckBz6kc4jRQGxqAzNkhT1XZIF0yPqyCQSnezWO3osMw1ewpjToO7q41E9.LvedWY"
+    },
+    "params": {},
+    "body": {
+      "JSON": {
+        "Email": "peter.gibbons@initech.com",
+        "Phone": "570-690-4150",
+        "Rating": "Hot",
+        "Title": "VP of Derp",
+        "FirstName": "Peter",
+        "LastName": "Gibbons",
+        "PostalCode": "94115",
+        "City": "east greenwich",
+        "Country": "USA",
+        "State": "California",
+        "Street": "19123 forest lane",
+        "Company": "Initech"
+      },
+      "XML": {},
+      "JSON_ARRAY": {},
+      "FORM": {}
+    },
+    "files": {}
+  },
+  "metadata": [
+    {
+      "jobId": 1
+    },
+    {
+      "jobId": 1
+    },
+    {
+      "jobId": 1
+    }
+  ],
+  "batched": false,
+  "statusCode": 200,
+  "destination": {
+    "Config": {
+      "initialAccessToken": "9f58ryaM64FZp0rg3jyDV4PB5",
+      "password": "hJCR557#uzFfD",
```

The values of initialAccessToken and password shouldn't be same as may be something tested. Please change these to something that looks like test-data (a suggestion is provided below)
```suggestion
      "initialAccessToken": "9f58ryaInitToken1343dsu73",
      "password": "myPa#swd123",
```

---

Sure, will do.